integration_tests/array_expr_01.py

```diff
@@ -0,0 +1,42 @@
+from ltypes import i32, f32, f64
+from numpy import empty, reshape
+
+def array_expr_01():
+    dim1: i32
+    dim2: i32
+    dim3: i32
+    dim1d: i32
+    i: i32
+    shape1d: i32[1] = empty(1, dtype=int)
+    shape3d: i32[3] = empty(3, dtype=int)
```

Comment:
I think we have to set these types to int32, where `int32` is imported from NumPy.
```suggestion
    shape1d: i32[1] = empty(1, dtype=int32)
    shape3d: i32[3] = empty(3, dtype=int32)
```